config/shared.maven.filters

```diff
@@ -61,6 +61,7 @@ shared.download_form.pdfurl=
 shared.ldapadmin.contextpath=/ldapadmin
 shared.ldapadmin.db=georchestra
 shared.ldapadmin.jdbcurl=jdbc:postgresql://@shared.psql.host@:@shared.psql.port@/@shared.ldapadmin.db@
+shared.ldapadmin.warnifuidmodified=true
```

Comment:
My mistake, this is not a real shared maven filter. It only belongs to ldapadmin.
=> should not be set in config/shared.maven.filters but in build_support/GenerateConfig.groovy